routes/auth/seller.js

```diff
@@ -27,18 +35,22 @@ router.get('/sellers', (req, res) => {
 });
 
 
-// Update information of a seller
+// Update seller profile
 router.put('/sellers', (req, res) => {
   const params = req.body;
-  sellerService.updateSeller(params)
-    .then(seller => { 
-      res.json({
-        success: true,
-        result: seller
-      });
-    }).catch(err => {
-      if (typeof(err) != 'string') {
-        console.error('Error /auth/seller/update/profile', err);
+  params.id = req.decoded.id;
+  params.email = req.decoded.email;
+  sellerController.updateSeller(params)
```

Comment:
Can we say `updateSeller(req.decoded)`?

Reply:
no params has all the fields e.g. new password. id and email cannot be updated so overriding here.